src/config.rs

```diff
@@ -97,6 +97,8 @@ pub struct Config {
     pub no_tld_query: bool,
     /// Force using TCP for DNS resolution
     pub use_vc: bool,
+    /// OpenBSDs lookup file bind option
+    pub lookup_file_bind: bool,
```

Comment:
This should be a vector. Presumably `Option<Vec<OpenBsdLookupOption>>` so that it's easy to tell when it's present.

Reply:
I would implement it as an ´´´Option<bool>´´´ if its ok for you. I do not see a reason for a Vector and a new type (or alias).

Reply:
Well, It's very unclear what it means. We're trying to make it easy to understand and map directives close to structure fields. I think it's better to use a vector. And make a convenience method that returns a bool.

I'm also not sure about custom values. Maybe we should keep them in the vector?

Another thing to consider if in any other system "lookup" directive allows a different set of values. Although, I didn't find any by quick googling.

Reply:
Ok, I get your point. I'm just confused about the Vector. It can only be true or false. So the vector will of size 1 at best. I could also use a ```Option<OpenBsdLookupOtion>```.
Don't get me wrong, I don't have anything against using a Vector, I just don't get why (maybe I'm missing something obvious though).
It makes sense for the ```search``` Field in the struct, but for a single bool I don't know.

Reply:
The line
```
lookup file bind something
```
Should yield:
```
lookup: Some(vec![File, Bind, Extra(String::from("Something"))]),
```
Then it can be turn into bool by a method:
```
fn openbsd_lookup_file_first(&self) -> bool {
    self.lookup.as_ref().map(|vec| {
        vec.find(File).unwrap_or(usize::MAX) < vec.find(Bind).unwrap_or(usize::MAX)
    }).unwrap_or_else(false)
}
```
This allows different semantics on non-openbsd systems. And provides more concise semantics of openbsd (is what I've written correct?)

Does it make more sense now?